cli/src/config/templates.toml

```diff
@@ -3,6 +3,8 @@ commit_summary = '''
 separate(" ",
   format_short_change_id(change_id),
   format_short_commit_id(commit_id),
+) ++ label("separator", " |") ++
+separate(" ",
```

Comment:
nit: if we need separator, I slightly prefer `" | "` than `" |"`.

Reply:
Do you like either of these? You also have to imagine what it would look like in black-and-white.

![image](https://github.com/martinvonz/jj/assets/4123047/da84f697-57b3-4132-bd1c-581fdd084508)

Reply:
In any case, I will also make the separator configurable (its own alias).

Reply:
The former (the `" | "` one) looks good to me. I feel double spaces `"  "` is also good.

If the reason of inserting separator is to distinguish `branches` from `description` without coloring, `"|"` with no spaces would be bad.

Reply:
Sounds good, thanks!

Reply:
Double spaces is an interesting idea. I don't think I immediately like it, but I'll try it out and see how it makes tests look. It has the advantage that we could use it regardless of whether there are any branches.